mbed_greentea/mbed_test_api.py

```diff
@@ -165,6 +166,27 @@ def run_command(cmd):
             return None
         return p
 
+    def run_htrun(cmd):
```

Comment:
Could you pull this and `run_command` out of this function?

Reply:
sure.